### DISCUSSION TITLE
Virtual text completion

### DISCUSSION THREAD

**Discussion by madflow** [Exafunction/codeium.nvim#139] [General]
Is "virtual text completion" like in the "vim" plugin not possible, because it just has not been implemented yet, or is it not possible from a technical standpoint?

https://github.com/Exafunction/codeium.nvim/issues/133

**Comment by KyrillGobber** [comment]
It's really weird that the vim plugin has virtual text completion, but the nvim one does not. I'm switching back to the vim one for this, but I really really like the cmp functionallity of the nvim plugin. So sad

**Reply by dimfeld** [reply]
I have this almost done in #241.
